RNI18n/RNI18n.h

```diff
@@ -7,8 +7,8 @@
 //
 
 #import <Foundation/Foundation.h>
-#import "RCTBridgeModule.h"
-#import "RCTLog.h"
+#import <React/RCTBridgeModule.h>
+#import <React/RCTLog.h>
```

Comment:
This is a breaking change, am I right? Does this work on RN <0.40?
If it's a breaking change we should bump the major version then.